client/my-sites/site-settings/theme-enhancements/index.jsx

```diff
@@ -0,0 +1,288 @@
+				checked={ this.isFieldTruthy( name ) }
+				disabled={ this.isFormPending() }
+				onChange={ this.handleToggle( name ) }>
+				<span>
+					{ label }
+				</span>
+			</FormToggle>
+		);
+	}
+
+	renderHeader() {
+		const {
+			submittingForm,
+			updatingSettings,
+			translate
+		} = this.props;
+		const formPending = this.isFormPending();
+
+		return (
+			<SectionHeader label={ translate( 'Theme Enhancements' ) }>
+				<Button
+					compact
+					primary
+					onClick={ this.onSubmitForm }
+					disabled={ formPending }
+				>
+					{ submittingForm || updatingSettings
+						? translate( 'Saving…' )
+						: translate( 'Save Settings' )
+					}
+				</Button>
+			</SectionHeader>
+		);
+	}
+
+	renderInfiniteScrollSettings() {
+		const {
+			selectedSiteId,
+			infiniteScrollModuleActive,
+			translate
+		} = this.props;
+		const formPending = this.isFormPending();
+
+		return (
+			<FormFieldset>
+				<div className="theme-enhancements__info-link-container">
+					<InfoPopover position={ 'left' }>
+						<ExternalLink href={ 'https://jetpack.com/support/infinite-scroll' } target="_blank">
+							{ translate( 'Learn more about Infinite Scroll' ) }
+						</ExternalLink>
+					</InfoPopover>
+				</div>
+
+				<JetpackModuleToggle
+					siteId={ selectedSiteId }
+					moduleSlug="infinite-scroll"
+					label={ translate( 'Add support for infinite scroll to your theme.' ) }
+					disabled={ formPending }
+					/>
+
+				{
+					infiniteScrollModuleActive && (
+						<div className="theme-enhancements__module-settings is-indented">
+							{
+								this.renderToggle( 'infinite_scroll', translate(
+									'Scroll infinitely (Shows 7 posts on each load)'
+								) )
+							}
+							{
+								this.renderToggle( 'infinite_scroll_google_analytics', translate(
+									'Track each infinite Scroll post load as a page view in Google Analytics'
+								) )
+							}
+						</div>
+					)
+				}
+			</FormFieldset>
+		);
+	}
+
+	renderMinilevenSettings() {
+		const {
+			selectedSiteId,
+			minilevenModuleActive,
+			translate
+		} = this.props;
+		const formPending = this.isFormPending();
+
+		return (
+			<FormFieldset>
+				<div className="theme-enhancements__info-link-container">
+					<InfoPopover position={ 'left' }>
+						<ExternalLink href={ 'https://jetpack.com/support/mobile-theme' } target="_blank">
+							{ translate( 'Learn more about Mobile Theme' ) }
+						</ExternalLink>
+					</InfoPopover>
+				</div>
+
+				<JetpackModuleToggle
+					siteId={ selectedSiteId }
+					moduleSlug="minileven"
+					label={ translate( 'Optimize your site with a phone-friendly theme.' ) }
+					disabled={ formPending }
+					/>
+
+				{
+					minilevenModuleActive && (
+						<div className="theme-enhancements__module-settings is-indented">
+							{
+								this.renderToggle( 'wp_mobile_excerpt', translate(
+									'Use excerpts instead of full posts on front page and archive pages'
```

Comment:
Alternate string suggestion: 
* `translate( 'Show excerpts on front page and on archive pages instead of full posts' )`&mdash; translations: **17**. 

[PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/jetpack-setings-theme-enhancements-card)